index.html

```diff
@@ -0,0 +1,214 @@
+                        </div>                    
+                    </div>
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion6.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">Contributing to Open-Source Projects as a New Python...</h2>
+                            <h2 class="sugAuthor">Real Python</h2>
+                            <h2 class="sugView">25 K view</h2>
+                        </div>                    
+                    </div>
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion7.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">Google Chrome Developer Tools Crash Course </h2>
+                            <h2 class="sugAuthor">Traversy Media</h2>
+                            <h2 class="sugView">254 K view</h2>
+                        </div>                    
+                    </div>
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion8.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">50 Dev tool tips and tricks:  Become an expert front end developer </h2>
+                            <h2 class="sugAuthor">That JS Dude</h2>
+                            <h2 class="sugView">45 K view</h2>
+                        </div>                    
+                    </div>
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion9.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">JavaScript Pro Tips - Code This, NOT That</h2>
+                            <h2 class="sugAuthor">Fireship</h2>
+                            <h2 class="sugView">1,1 Mn view</h2>
+                        </div>                    
+                    </div>
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion10.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">Debugging JavaScript - Beginner to Advanced in One Video</h2>
+                            <h2 class="sugAuthor">youwebdev</h2>
+                            <h2 class="sugView">105 K view</h2>
+                        </div>                    
+                    </div>
+                </div>
+                <h4 class="more">SHOW MORE</h4>       
+            </aside>
+        </main> 
+
+        <footer>
+           <img src="images/footer.png" alt="">
```

Comment:
- [ ] [IMPORTANT!] You should never just take a screenshot of a whole page are and use it as an actual element. 
Imagine how frustrating it would be for a user to try and select any text from the footer or to click one of its links.